src/components/rows/grouped-items.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Col, Row } from 'reactstrap';
+import '../home.css';
+
+//This component encapsulates items on productsavailable.js
+export default class GroupedItems extends Component {
+	render() {
+		return (
+              <Container>
+                     <Row>
+                        <Col>
+                           <ul>
+                              <h6>{this.props.heading}</h6>
+                              <li><Link to={this.props.link1} style={{color: 'DarkGray'}}>{this.props.item1}</Link></li>
+                              <li><Link to={this.props.link2} style={{color: 'DarkGray'}}>{this.props.item2}</Link></li>
+                              <li><Link to={this.props.link3} style={{color: 'DarkGray'}}>{this.props.item3}</Link></li>
+                              <li><Link to={this.props.link4} style={{color: 'DarkGray'}}>{this.props.item4}</Link></li>
+                              <li><Link to={this.props.link5} style={{color: 'DarkGray'}}>{this.props.item5}</Link></li>
+                           </ul>              
```

Comment:
this component should take a prop called `items` which is an array of objects that looks like
```
[{ link: 'exampleLink', name: 'exampleName'}, ...];
```
Then you should map over the items array to render the items
```
this.props.items.map(item => (<li><Link to={item.link} className={'group-items'}>{item.name}</Link></li>));
```